spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/sql/TestSelect.java

```diff
@@ -49,8 +49,8 @@ public TestSelect(String catalogName, String implementation, Map<String, String>
 
   @Before
   public void createTables() {
-    sql("CREATE TABLE %s (id bigint, data string, float float) USING iceberg", tableName);
-    sql("INSERT INTO %s VALUES (1, 'a', 1.0), (2, 'b', 2.0), (3, 'c', float('NaN'))", tableName);
+    sql("CREATE TABLE %s (id bigint, data string, float float, binary binary) USING iceberg", tableName);
+    sql("INSERT INTO %s VALUES (1, 'a', 1.0, X''), (2, 'b', 2.0, X'11'), (3, 'c', float('NaN'), X'1111')", tableName);
```

Comment:
Rather than repurposing old test cases, can you create new ones? We want to avoid mixing tests together in confusing ways that look like other failures.

Reply:
I'd also add that repurposing existing test cases makes it harder for people who maintain forks and might not cherry-pick every commit.

Reply:
Thanks for your review, I have modified the code